ENVParser/EnvFile.cs

```diff
@@ -0,0 +1,580 @@
+            ModelFarClip = reader.ReadSingle();
+            EnableFog = reader.ReadBoolean();
+            EnableAmbientFog = reader.ReadBoolean();
+            DisableFog = reader.ReadBoolean();
+            ToggleFogCameraPlane = reader.ReadBoolean();
+            FogStartDistance = reader.ReadSingle();
+            FogEndDistance = reader.ReadSingle();
+            FogRed = reader.ReadSingle();
+            FogGreen = reader.ReadSingle();
+            FogBlue = reader.ReadSingle();
+            FogAlpha = reader.ReadSingle();
+            EnableFloorFog = reader.ReadBoolean();
+            FloorFogStartingHeight = reader.ReadSingle();
+            FloorFogEndHeight = reader.ReadSingle();
+            FloorFogRed = reader.ReadSingle();
+            FloorFogGreen = reader.ReadSingle();
+            FloorFogBlue = reader.ReadSingle();
+            FloorFogOpacity = reader.ReadSingle();
+            EnableGraphicOutput = reader.ReadBoolean();
+            EnableBloom = reader.ReadBoolean();
+            EnableGlare = reader.ReadBoolean();
+            Field1CC = reader.ReadUInt32();
+            Field1D0 = reader.ReadUInt32();
+            BloomAmount = reader.ReadSingle();
+            BloomDetail = reader.ReadSingle();
+            BloomWhiteLevel = reader.ReadSingle();
+            BloomDarkLevel = reader.ReadSingle();
+            GlareSensitivity = reader.ReadSingle();
+            SceneWhiteLevels = reader.ReadSingle();
+            SceneDarkLevels = reader.ReadSingle();
+            Field1F0 = reader.ReadSingle();
+            Field1F4 = reader.ReadSingle();
+            Field1F8 = reader.ReadSingle();
+            Field1FC = reader.ReadUInt32();
+            Field200 = reader.ReadSingle();
+            Field204 = reader.ReadSingle();
+            Field208 = reader.ReadUInt32();
+            Field20C = reader.ReadUInt32();
+            Field210 = reader.ReadSingle();
+            Field214 = reader.ReadSingle();
+            RedColourBoost = reader.ReadSingle();
+            GreenColourBoost = reader.ReadSingle();
+            BlueColourBoost = reader.ReadSingle();
+            Field224 = reader.ReadSingle();
+            Field228 = reader.ReadSingle();
+            Field22C = reader.ReadSingle();
+            Field230 = reader.ReadSingle();
+            Field234 = reader.ReadSingle();
+            Field238 = reader.ReadSingle();
+            Field23C = reader.ReadSingle();
+            GlareLength = reader.ReadSingle();
+            GlareChromaticAberration = reader.ReadSingle();
+            GlareDirection = reader.ReadSingle();
+            GlareMode = reader.ReadUInt32();
+            Field250 = reader.ReadBoolean();
+            Field251 = reader.ReadUInt32();
+            Field255 = reader.ReadSingle();
+            Field259 = reader.ReadSingle();
+            Field25D = reader.ReadSingle();
+            EnableDOF = reader.ReadBoolean();
+            DOF_FocalPlane = reader.ReadSingle();
```

Comment:
Would it make sense to group the `DOF_`, `SSAO_`, etc. fields into their own objects, to keep the EnvFile class a bit cleaner + the property names shorter?

Reply:
For the named fields, potentially, however, I wouldn't want to _overstate__ the meaning of the fields, since I used either the tempalte names or the research we did in the discord thread, meaning its based on an assumption

Reply:
True, though it'd be easy enough to undo if the assumptions get overturned later. My entire thinking is to cut down on the number of direct properties of EnvFile by grouping them wherever it makes sense.